content/en/docs/tasks/administer-cluster/access-cluster-api.md

```diff
@@ -161,28 +161,38 @@ Kubernetes officially supports client libraries for [Go](#go-client) and
 
 #### Go client
 
-* To get the library, run the following command: `go get k8s.io/client-go/<version number>/kubernetes` See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go) to see which versions are supported.
-* Write an application atop of the client-go clients. Note that client-go defines its own API objects, so if needed, please import API definitions from client-go rather than from the main repository, e.g., `import "k8s.io/client-go/1.4/pkg/api/v1"` is correct.
+* To get the library, run the following command: `go get k8s.io/client-go@kubernetes-<kubernetes-version-number>` See [https://github.com/kubernetes/client-go](https://github.com/kubernetes/client-go) to see which versions are supported.
+* Write an application atop of the client-go clients. Note that client-go defines its own API objects, so if needed, import API definitions from client-go rather than from the main repository. For example, `import "k8s.io/client-go/kubernetes"` is correct.
+
+{{< note >}}
+
+According to the [README](https://github.com/kubernetes/client-go/blob/master/README.md), the client-go responsitory is still a mirror of [https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/client-go](https://github.com/kubernetes/kubernetes/tree/master/staging/src/k8s.io/client-go), the code development is in the staging area. Since Kubernetes 1.8 release, the client-go library syncs the Kubernetes version tags to client-go when syncing the code from the staging area, prefixed with "kubernetes-". For example, if you check out the kubernetes-1.15.3 tag in client-go, the code you get is exactly the same as you check out the v1.15.3 tag in Kubernetes, and change directory to `staging/src/k8s.io/client-go`. 
```

Comment:
I wonder if this explanation is needed in the page. I do want to have the information about version numbering clarified in this PR, so that reviewers can make sure it's the right change to make.

Reply:
OK. I got it.